### DISCUSSION TITLE
The location of the markdown button and the code button

### DISCUSSION THREAD

**Discussion by dododo0611** [microsoft/vscode-jupyter#10585] [Questions and Answers]
I don't know why the code button goes to the right side of the markdown button  on the toolbar. It changes the style and I am not used to it. Is there any way to change the order? Thanks!
Before:
<img width="747" alt="Screen Shot 2022-06-27 at 18 25 56" src="https://user-images.githubusercontent.com/63335151/175920875-2208dbc4-101a-46fd-a8b8-49eccacd5496.png">
Now:
<img width="487" alt="Screen Shot 2022-06-27 at 18 26 43" src="https://user-images.githubusercontent.com/63335151/175921039-aad0a912-153e-4f62-823f-226d5eb3871f.png">

**Comment by rchiodo** [comment]
They're likely sorted alphabetically within a group. So now that the 'Code' button was translated, it moved. Hopefully the translation of 'Markdown' will change the order back.
